src/tutorial/untangled_tutorial/H_Server_Interactions.cljs

```diff
@@ -123,29 +383,84 @@
   where the tx that failed indicates the app state is invalid...additional network interactions are probably all going to
   fail. What you want to do is trigger some kind of state reload to restore sanity.
 
-  #### Updating an existing item
+  #### Remote reads after a mutation
 
-  TODO: Just like Om
+  In Om, you can list properties after your mutation to indicate re-renders. You can force them to be remote reads by
+  quoting them. All of this requires complex logic in your parser to compare flags on the AST, process the resulting
+  query in send (e.g. via process-roots), etc. It is more flexible, but the very common case can be made a lot more direct.
+
+  ```
+  (om/transact! this '[(app/f) ':thing]) ; run mutation and re-read thing on the server...
+  ; BUT YOU implement the logic to make sure it is understood that way!
+  ```
+
+  In Untangled, follow-on keywords are always local re-render reads, and nothing more:
+
+  ```
+  (om/transact! this '[(app/f) :thing])
+  ; Om and Untangled: Do mutation, and re-render anything that has :thing in a query
+  ```
+
+  Instead, we supply access to the internal mutation we use to queue loads, so that remote reads are simple and explicit:
+
+  ```
+  ; Do mutation, then run a remote read of the given query, along with a post-mutation
+  ; to alter app state when the load is complete.
+  (om/transact! this `[(app/f) (untangled/load {:query ~(om/get-query Thing) :post-mutation after-load-sym}])
+  ```
+
+  Of course, you can (and *should*) use syntax quoting to embed a query from (om/get-query) so that normalization works,
+  and the post-mutation can be used to deal with the fact that other parts of the UI may want to, for example, point
+  to this newly-loaded thing. The `after-load-sym` is a symbol (e.g. dispatch key to the mutate multimethod). The multi-method
+  is guaranteed to be called with the app state in the environment, but no other Om environment items are ensured at
+  the moment.
+
+  ### Writing Server Mutations
+
+  Server-side mutations in Untangled are written the same way as in Om. A mutation returns a map with a key `:action`
+  and a function of no variables as a value. That function then creates, updates, and/or deletes data from the data
+  storage passed in the `env` parameter to the mutation.
 
   #### New item creation – Temporary IDs
 
-  TODO: Similar to Om, but `{ :tempids tempidmap }` can be returned from `:action` of server mutation and it will
-  just work. The plumbing is pre-written.
+  Om has a built in function `tempid` that will generate an om-specific temporary ID. This allows the normalization
```

Comment:
Should say something about how `om/tempid` must be called from the UI (because mutations are triggered more than once, which would cause inconsistencies if you used `tempid` within the mutation code)